src/index.html

```diff
@@ -22,6 +22,394 @@
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div
+        class="container"
+        data-qa="card"
+      >
+        <img
+          class="container__image"
+          src="images/imac.jpeg"
+          alt="Apple Device(imac)"
+        />
+        <div class="container__product-info">
+          <h3 class="product-info-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="product-info-text">Product code: 195434</p>
+        </div>
+        <div class="container__stars-reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="review-text">Reviews: 5</p>
+        </div>
+        <div class="container__product-price">
+          <p class="price-text">Price:</p>
+          <span class="cost">$2,199</span>
+        </div>
+        <a
+          data-qa="card-hover"
+          href="#"
+          class="container__btn"
+        >
+          BUY
+        </a>
```

Comment:
The block of code from lines 110 to 145 is repeated multiple times with the same content. It would be better to create a template and use a loop to generate these elements if possible.